service/src/main/java/org/cbioportal/service/impl/SampleServiceImpl.java

```diff
@@ -174,31 +174,51 @@ public BaseMeta fetchMetaSamples(List<String> sampleListIds) {
         return sampleRepository.getSamplesByInternalIds(internalIds);
     }
 
-    private void processSamples(List<Sample> samples, String projection) {
+    private Boolean isProfiledWithSV(MolecularProfile p) {
+        return (p.getMolecularAlterationType().equals(MolecularProfile.MolecularAlterationType.STRUCTURAL_VARIANT) ||
+                hasFusionsAsMutations(p));
```

Comment:
Curious why you added hasFusionsAsMutations in this clause.

Reply:
so that the sample status `isProfiledWithSV` can be set to true in the mskarcher cases

the lambda function was a little.. ugly with everything included right there so I moved it to a function but I can move it back here instead if that's preferred

Reply:
I just noticed you are calling both below, so didn't think it was necessary.